sympy/core/relational.py

```diff
@@ -186,6 +190,10 @@ def _eval_simplify(self, ratio, measure):
             if v is not None:
                 r = r.func._eval_relation(v, S.Zero)
 
+            numer, denom = dif.as_numer_denom()
+            if numer.is_zero is False and numer.is_number is True:
```

Comment:
I think it's sufficient to say 'numer.is_zero is False' b/c even if it is symbolic (e.g. var('x', positive=True)) it should evaluate.

Regarding @asmeurer comment about this being wrong for an inequality: you are only doing valid operations that don't change the sign of the relational, `a rel_op b ==> a - b rel_op 0`. It is the solving for a particular variable that can cause problems b/c that may require dividing though by a quantity whose sign is unknown. But you aren't doing that here.